lib/gc/Dialect/Linalgx/LinalgOps.cpp.inc

```diff
@@ -0,0 +1,544 @@
+//===- LinalgOps.cpp - Implementation of the linalg operations ------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file is a partial copy of mlir/lib/Dialect/Linalg/IR/LinalgOps.cpp
```

Comment:
Is it possible to directly use the original instead?

Reply:
Has not found another way. The file cannot be included after MLIR is built and the class/helper functions are static and cannot be used in other files.